protostar/compiler/project_cairo_path_builder.py

```diff
@@ -1,38 +1,18 @@
-import os
 from pathlib import Path
 
-from protostar.configuration_file import ConfigurationFile
-
 
 class ProjectCairoPathBuilder:
```

Comment:
Can we remove this class? It seems too small to abstract it

Reply:
This logic is used in various places in the application despite being "lite".

Reply:
Resolving. I don't want to mix refactors with functionality changes if possible.

Reply:
Can you make a task to remove it?

Reply:
https://github.com/software-mansion/protostar/issues/1207